ckanext/validation/jobs.py

```diff
@@ -85,11 +90,7 @@ def run_validation_job(resource):
 
     schema = resource.get(u'schema')
     if schema and isinstance(schema, string_types):
-        if schema.startswith('http'):
-            r = requests.get(schema)
-            schema = r.json()
-        else:
-            schema = json.loads(schema)
+        schema = json.loads(schema)
```

Comment:
just wondering if this will break on legacy data, where schema is a url?

Reply:
You are right, I've restored this for legacy data and added a unit test.